### DISCUSSION TITLE
Where are my inscriptions ?

### DISCUSSION THREAD

**Discussion by GaloisField2718** [ordinals/ord#1835] [Q&A]
Hello everyone, 

For a first try on ordinals I tried this command : 
`cargo run wallet inscribe inscription/galoisfield.btc.txt`

Everything goes well and I got this output : 
```bash
    Finished dev [unoptimized + debuginfo] target(s) in 1.94s
     Running `target/debug/ord wallet inscribe inscription/galoisfield.btc.txt`
{
  "commit": "ebcd883015295586d9db921e649fdc04a02476827591103aa28e85ec3d36d903",
  "inscription": "23aef7fa2df44a0192dde9ce14bee594ece7890c701820ed7c727485f154e4e0i0",
  "reveal": "23aef7fa2df44a0192dde9ce14bee594ece7890c701820ed7c727485f154e4e0",
  "fees": 311
}
```

But when I look to find this inscription on the ordinals.com/inscription website I don't see anything. 

Moreover, 

I tried : `ord wallet inscriptions` and `ord wallet sats` and `ord wallet outputs` 
but every commands out with an empty array : []

What can I do ? Are they lost ? 

I tried another .txt file and the same happening with still empty array. 

I don't really understand each things and as in the rust code I don't really find something related to this "issue". 

It might be something which I didn't understand.

**Comment by satoshi0770** [comment]
You can search for your reveal tx on mempool.space
In your example, looks like its still pending and has not been confirmed into a block yet.

https://mempool.space/tx/23aef7fa2df44a0192dde9ce14bee594ece7890c701820ed7c727485f154e4e0

**Reply by GaloisField2718** [reply]
It looks weird... 
The commit number gives me this transaction : https://mempool.space/tx/ebcd883015295586d9db921e649fdc04a02476827591103aa28e85ec3d36d903 
And this is validated. It's more logic also by the next result : 
`  {
    "transaction": "ebcd883015295586d9db921e649fdc04a02476827591103aa28e85ec3d36d903",
    "confirmations": 54
  }`
So I don't understand what is reveal transaction. 
Do you know what is it ?

**Reply by GaloisField2718** [reply]
P.S. : I downloaded Sparrow wallet I will check on it but can we mint with Sparrow Wallet ?

**Reply by satoshi0770** [reply]
cannot inscribe with sparrow, sparrow just to help with ui to see whats going on.
when inscribing there is 2 transactions that happen.
first is commit tx
commit takes a cardinal utxo
uses that as input, and creates 2 outputs, output1 and output2
thats 1 transaction. that is called the commit tx.

now the second transaction, it takes the first output of the first transaction,
and uses that as an input, and adds witness data in the input field, and has an output of 10k sats. and that utxo created is where the inscription is.

the 2nd output of the first transaction is the change from your cardinal utxo that u started with.

hope that explains it.

**Reply by satoshi0770** [reply]
also, feel free to jump in the ordinals discord and hop on voice, usually alot of helpful folks in there can help out. and or ask in the tech support channel.

**Reply by GaloisField2718** [reply]
Ok yeah really more clear. Only utxo for me is Unspend Transaction Outputs and I don’t see what the link with our work here. Maybe I just have to take information about this object in general and not in this specific case.

**Reply by GaloisField2718** [reply]
> also, feel free to jump in the ordinals discord and hop on voice, usually alot of helpful folks in there can help out. and or ask in the tech support channel.

Yeah yeah I’m trying to be active on the dev-nerds channel and trying to help somebody with basic issues 😄

**Comment by GaloisField2718** [comment]
Aaaaaah !!! ah yes indeed !! Oh! 
But so I have a question : why have I this result : 
```bash
 ord wallet transactions                             
[
  {
    "transaction": "e3d3a0a46dedb00cf51165b256b0ba0807b9321a68f3a4fb0175a761b57c460c",
    "confirmations": 1117
  },
  {
    "transaction": "15be211472ba8322e865dc9b15ade5fde1ec287078f49a01b93a0e8077b76067",
    "confirmations": 60
  },
  {
    "transaction": "ebcd883015295586d9db921e649fdc04a02476827591103aa28e85ec3d36d903",
    "confirmations": 54
  },
  {
    "transaction": "ebcd883015295586d9db921e649fdc04a02476827591103aa28e85ec3d36d903",
    "confirmations": 54
  }
``` 
Last one ebcd... seems to be under 54 confirmations and also why have I 2 transactions ? 
Maybe it's a dumb question but I don't know why ^^'

**Reply by satoshi0770** [reply]
not sure,
can always import the wallet to sparrow, to have a better gui of whats going on

**Reply by GaloisField2718** [reply]
Yes thanks I will try you’re right

**Comment by GaloisField2718** [comment]
thanks @casey for moving

**Reply by GaloisField2718** [reply]
I could put other questions ^^
